packages/server/lib/controllers/v1/logs/searchLogs.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from 'zod';
+import { asyncWrapper } from '../../../utils/asyncWrapper.js';
+import { requireEmptyQuery, zodErrorToHTTP } from '../../../utils/validation.js';
+import type { SearchLogs } from '@nangohq/types';
+import { model, envs } from '@nangohq/logs';
+
+const validation = z
+    .object({
+        limit: z.number().optional().default(100)
+    })
+    .strict();
+
+export const searchLogs = asyncWrapper<SearchLogs>(async (req, res) => {
+    if (!envs.NANGO_LOGS_ENABLED) {
+        res.status(422).send({ error: { code: 'feature_disabled' } });
+        return;
+    }
+
+    const emptyQuery = requireEmptyQuery(req, { withEnv: true });
```

Comment:
not sure why `requireEmptyQuery` is a function where below we do the validation of the body inline, or do you mean to use `requireEmptyBody` below?

Reply:
`requireEmptyQuery` is a utils to avoid all the endpoints that have no query params to rewrite this logic. 
the body is not empty and specific to this endpoint so no shared logic to reuse.
Don't know if that makes sense